<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrightfieldLedDirectManipDialog</class>
 <widget class="QDialog" name="BrightfieldLedDirectManipDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>101</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>brightfield_led Module Direct Manipulator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="enabledToggle">
       <property name="text">
        <string>Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="powerSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="powerSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>enabledToggle</sender>
   <signal>toggled(bool)</signal>
   <receiver>BrightfieldLedDirectManipDialog</receiver>
   <slot>enabledToggledSlot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>powerSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrightfieldLedDirectManipDialog</receiver>
   <slot>powerSpinBoxValueChangedSlot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>powerSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrightfieldLedDirectManipDialog</receiver>
   <slot>powerSliderValueChangedSlot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enabledToggledSlot(bool)</slot>
  <slot>powerSliderValueChangedSlot(int)</slot>
  <slot>powerSpinBoxValueChangedSlot(int)</slot>
 </slots>
</ui>
